lib/key-encryption.ts

```diff
@@ -0,0 +1,133 @@
+import { CryptoUtil } from "./crypto-util.js";
+import Converters = CryptoUtil.Converters;
+
+import webcrypto from "./get-crypto.js";
+
+const iterations = 223978;
+
+export interface IEncryptedJSON {
+    iv: string;
+    salt: string;
+    ciphertext: string;
+}
+
+interface ISaltIV {
+    iv: Uint8Array;
+    salt: Uint8Array;
+}
+
+export const KeyEncryption = {
```

Comment:
You can make this a class and have the method static, that is the more common way I guess, that way you can mark them as public/private as you need.

Reply:
Will do, thanks.

Reply:
done